conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -13,7 +13,7 @@ docker info
 config=$(cat <<CONDARC
 
 channels:
-{%- for channel in channels.get('sources', []) | sort %}
+{%- for channel in channels.get('sources', []) %}
```

Comment:
Seems I'm changing something you had added [before]( https://github.com/conda-forge/conda-smithy/pull/19/files#diff-00e77a6004c558c75ab3d1be6bfa56d9L16 ), @ericdill . Would be great to hear your feedback on this.